microsoft-365/admin/manage/change-address-contact-and-more.md

```diff
@@ -52,6 +52,9 @@ To change other information on your company's profile page:
 2. On the **Organization profile** tab, select **Organization information**.
 3. Update your organization's information, then select **Save changes**. Be sure to fill in all required fields marked with an * to enable saving your changes.
 
+> [!NOTE]
+> SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize the document libraries to your PC you will encounter error messages when attempting to do anything within the synchronized document libraries (such as creating folders or renaming documents).
```

Comment:
Please replace paragraph with:

SharePoint Online and OneDrive have a 256-character limit on Windows PCs. When you synchronize document libraries to your PC, you will receive error messages when you attempt to do anything within the synchronized document libraries, such as creating folders or renaming documents.